assets/app/view/invite_game.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'game_manager'
+require 'view/game_row'
+
+module View
+  class InviteGame < Snabberb::Component
+    include GameManager
+
+    needs :user
+    needs :refreshing, default: nil, store: true
```

Comment:
why do you need this?

Reply:
We don't. It's an artifact that survived from a previous attempt